src/NLog.Web.AspNetCore/LayoutRenderers/AspNetConnectionHttpTransportTypeLayoutRenderer.cs

```diff
@@ -0,0 +1,52 @@
+﻿#if ASP_NET_CORE3
+using Microsoft.AspNetCore.Http.Connections.Features;
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// Specifies transports that the client can use to send HTTP requests.
+    ///
+    /// This enumeration supports a bitwise combination of its member values.
+    ///
+    /// None
+    /// WebSockets
+    /// ServerSentEvents
+    /// LongPolling
+    ///
+    /// </summary>
+    [LayoutRenderer("aspnet-connection-http-transport-typr")]
```

Comment:
Can `aspnet-connection` remain `aspnet-request` ? (And fix `-typr` so it becomes `-type`)